alt-tab-macos/logic/TrackedWindow.swift

```diff
@@ -42,4 +42,17 @@ class TrackedWindow {
         }
         axWindow?.focus(id)
     }
+
+    func close() {
+        if axWindow == nil {
+            axWindow = id.AXUIElementOfOtherSpaceWindow(ownerPid)
+        }
+        axWindow?.close()
+    }
```

Comment:
What about adding the following logic in a getter method to avoid duplication?

```swift
if axWindow == nil {
    axWindow = id.AXUIElementOfOtherSpaceWindow(ownerPid)
}
```

Reply:
Good idea, I'll add an var `getAxWindow` with the getter :)